T2MDCli/CliOptions.cs

```diff
@@ -42,6 +42,15 @@ public class CliOptions
         )]
         public bool ListBoardsOnly { get; set; } = false;
 
+        [Option(
+            "boards-to-include",
+            Required = false,
+            HelpText = "If specified, limits which boards are backed up.\n"
+                       + "Board names can be selected from the 'list-boards-only' operation.\n"
+                       + "Board names are comma separated ','"
+        )]
+        public string BoardsToInclude { get; set; } = "";
```

Comment:
Love it! Thanks for contributing. Can you change this to be an `IEnumerable<string>` like `ReplaceLinksDestinationBoardsToExclude` below? It makes the input format more flexible and will allow removing the `.Split(',')` in the main file.